CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Pinned typescript to ~4.7.x to address typing incompatibility issues
     discussed in https://github.com/knex/knex/pull/5279
   - Update generate-ts-build-cache script to always install root project dependencies
+- **CUMULUS-2913**
+  - Changed `process-dead-letter-archive` lambda to put messages from S3 dead
+    letter archive that fail to process to new S3 location.
```

Comment:
Please update doc about the location of failed messages
https://github.com/nasa/cumulus/blob/master/docs/features/dead_letter_archive.md

Reply:
https://github.com/nasa/cumulus/pull/3099/commits/a738f3f1cf4a78d1454c73c6654d2d47938d341a#diff-e8fac0f877b4264f513f4fb2d7a740717351d8252b67c3e1605c67c900c2beb1R19 Thanks for the reminder!